tests/jerry/es.next/promise-all-iterator.js

```diff
@@ -75,9 +75,9 @@ var rejects = Promise.all(createIterable([
 fulfills.then(result => { assert (result + "" === "foo,bar"); });
 rejects.catch(result => { assert (result === "baz"); });
 
-var closed = false;
+var closed = true;
 delete Promise.resolve;
-Promise.all(createIterable([1,2,3], {'return': function () { closed = true; }}));
+Promise.all(createIterable([1,2,3], {'return': function () { closed = false; }}));
```

Comment:
Was this wrong before?

Reply:
Nothing, but with the newest standard the **get resolve** is called earlier  and doesn't change closed bool value